xyz.stuerz.BilligSweeper.yml

```
@@ -0,0 +1,27 @@
+app-id: xyz.stuerz.BilligSweeper
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: billig-sweeper
+rename-desktop-file: billig-sweeper.desktop
+rename-icon: billig-sweeper
+finish-args:
+    - --socket=x11
+    - --device=dri
+    - --share=ipc
+
+modules:
+  - name: billig-sweeper
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/riscygeek/billig-sweeper/archive/refs/tags/v1.0.tar.gz
+        sha256: 27d79a341c5b15041bd4d0ff135da19c25460f014af1551a3d3421d000eb19e1
+    builddir: true
+  - name: billig-sweeper-metainfo
+    buildsystem: simple
+    sources:
+      - type: file
+        path: xyz.stuerz.BilligSweeper.metainfo.xml
+    build-commands:
+      - install -vDm644 -t /app/share/metainfo/ xyz.stuerz.BilligSweeper.metainfo.xml
```

one more time

```suggestion
    builddir: true
    post-install:
      - install -vDm644 -t /app/share/metainfo/ ../xyz.stuerz.BilligSweeper.metainfo.xml
    sources:
      - type: archive
        url: https://github.com/riscygeek/billig-sweeper/archive/refs/tags/v1.0.tar.gz
        sha256: 27d79a341c5b15041bd4d0ff135da19c25460f014af1551a3d3421d000eb19e1
      - type: file
        path: xyz.stuerz.BilligSweeper.metainfo.xml
```

this is the way.

---

This worked.